conf/consumer-connector.config/addresses.json

```
@@ -3,8 +3,8 @@
   "name": "consumer.edc.think-it.io",
   "addresses": {
     "default": "http://localhost:19191/api",
-    "data": "http://localhost:19193/api/v1/data",
-    "ids": "http://consumer-connector:9194/api/v1/ids",
+    "data": "http://localhost:19193/management",
+    "ids": "http://consumer-connector:9194/protocol",
```

```suggestion
    "management": "http://localhost:19193/management",
    "protocol": "http://consumer-connector:9194/protocol",
```

FYI: this was handled in another `TS` PR, so you can also wait until it's merged and then rebase